### ISSUE TITLE
Per-meeting statistics

### ISSUE THREAD

**Issue by lost1227** [momentumfrc/attendance-manager#27]
@FireShredder24 Wants a way to be able to see how many students attended a given meeting.

**Comment by lost1227** [comment]
```sql
SELECT DATE(checkin_date) AS date, COUNT(DISTINCT student_id) AS student_count
FROM attendance_sessions
GROUP BY date
ORDER BY date DESC;
```

**Comment by lost1227** [comment]
Closed by #29
